src/Manager/DoctrineUserManager.php

```diff
@@ -54,8 +54,11 @@ public function get(string $class, string $id)
     public function getClass($user): string
     {
         /** @var object $user */
-        /** @var class-string $class */
-        $class = ClassUtils::getClass($user);
```

Comment:
Why did you make that change?

Reply:
I thought ClassUtils was removed of `doctrine/common`, because I got the error it was not defined anymore
But it's slightly more complicated than that
In fact, with upper versions of doctrine/orm we have no dependency to `doctrine/common` anymore, which is why the class doest not exists
I just tried (locally) to add that dependency explicitely, and it seems to work !

Thanks for challenging :wink:

Reply:
https://github.com/yokai-php/security-token-bundle/pull/58/commits/21f72b78a1098f9708fb46b71681eb01a41531a2